src/pages/our_ambassadors.astro

```diff
@@ -392,6 +392,20 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         <a href="https://nf-co.re/governance#core-team" target="_blank">core team</a>.
       </AmbassadorCard>
 
+      <AmbassadorCard
+        name="Jeferyd Yepes"
+        img="jeferyd.jpg"
+        country="co"
```

Comment:
Your country of residence is Switzerland, right? In this case, the code here should be `ch`

```suggestion
        country="ch"
```